task/common/machine/script.go

```diff
@@ -102,6 +102,8 @@ fi
 
 rclone copy "$RCLONE_REMOTE/data" /tmp/tpi-task
 
+echo y | /etc/profile.d/install-driver-prompt.sh # for GCP GPU machines
```

Comment:
```suggestion
yes | /etc/profile.d/install-driver-prompt.sh # for GCP GPU machines
```

Yes because... why not?